app/views/local_transaction/devolved_administration_service.html.erb

```diff
@@ -0,0 +1,25 @@
+<%= render layout: 'shared/base_page', locals: {
+  title: @publication.title,
+  publication: @publication,
+  edition: @edition,
+} do %>
+  <div class="interaction">
+    <p class="govuk-body">
+        <%= t('formats.local_transaction.info_on_country_website', country_name: @country_name) %>
+    </p>
+    <div class="local-authority-result"
+         data-module="auto-track-event"
+         data-track-category="postcodeSearch:local_transaction"
+         data-track-action="postcodeResultShown:devolvedAdministrationService"
+         data-track-label=<%= @country_name %>>
```

Comment:
```suggestion
         data-track-label="<%= @country_name %>">
```

I assume you just missed this, it wouldn't matter if it wasn't for "Northern Ireland"

Reply:
thanks! I thought the syntax seemed strange with the ""! Have added back in :)